### PR TITLE
Update developer documentation

### PR DESCRIPTION
**Pull request by miha-plesko**
Developer documentation i.e. documentation about how to develop new recipe has not been complete. With this commit, we fill in the new environment variables description that were not described before and notify developer that she can connect to the container and manually experiment there. Also, we add a section that describes how to add package README easily.

Since we've just discovered that adding `--privileged` flag to the container run command decreases unikenrel compose time significantly as it enables KVM, we document this as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,11 @@ is a very simple task, but you have to wait quite some time.*
 Once having it on your machine, you can run it with:
 ```bash
 $ mkdir ./result
-$ docker run -it --volume="$PWD/result:/result" mikelangelo/capstan-packages
+$ docker run -it --priviledged --volume="$PWD/result:/result" mikelangelo/capstan-packages
```
Comment (gberginc) [inline README.md]: Replace with `--privileged` (no `d` in the middle).. Also in other lines below.

File: README.md [hunk 2]
```diff
@@ -15,8 +15,11 @@ is a very simple task, but you have to wait quite some time.*
...
 ```
+*NOTE: the `--priviledged` flag is needed in order to enable KVM inside container. Container will
+still work without the flag, but recipe tests will last longer.*
```
Comment (gberginc) [inline README.md]: Should you have two `**` at the end as well?

File: doc/recipes.md [hunk 3]
```diff
@@ -10,32 +10,41 @@ recipes
...
 Container's main process (implemented in [capstan-packages.py](../docker_files/capstan-packages.py))
 runs the `build.sh` script from within the recipe directory and provides following environment
 variables to it:
 
 | ENV | EXAMPLE VALUE | DESCRIPTION |
 |-----|---------------|-------------|
-| `RECIPE_DIR` | /recipes/eu.mikelangelo-project.osv.nfs | recipe directory |
-| `PACKAGE_RESULT_DIR` | /results/eu.mikelangelo-project.osv.nfs | directory where build result must appear |
-| `PACKAGE_NAME` | eu.mikelangelo-project.osv.nfs | same as directory name of recipe |
+| `RECIPE_DIR` | /recipes/osv.nfs | recipe directory |
```
Comment (gberginc) [inline doc/recipes.md]: Are these variables ordered in some sort? I believer the first two are most important, however a lexicographical order would be more natural now that there are many.

**Review by gberginc — CHANGES_REQUESTED** [review]
In the commit message you have `unikenrel`.

File: README.md (hunk 1 above)
Comment (miha-plesko) [reply README.md]: Oh my!

File: README.md (hunk 2 above)
Comment (miha-plesko) [reply README.md]: No, why? Lines 20 and 21 are meant to be italics so they are wrapped between two single `*`.

File: doc/recipes.md (hunk 3 above)
Comment (miha-plesko) [reply doc/recipes.md]: Good point, done.

**Review by miha-plesko — COMMENTED** [review]
Thanks for the review, I've amend-commited in order to also fix typo in commit message.

**Review by gberginc — APPROVED** [review]
Thanks!

**Pull request merged**